assembl/static2/css/components/text.scss

```diff
@@ -95,6 +99,32 @@
   font-size: $title5-size;
 }
 
+.collapsed-title {
+  background-color: $first-color-light;
```

Comment:
There are 2 main colors declined with the suffix `light` because in the mock up Roxana used an opacity